---
layout: page
title: Research
---


<h3 style="color:#3182bd"> Statistical and Computational Methods </h3>

<h4> Single-Cell Genomics </h4>
<font size="2"> (Under Construction) </font>
<h4> Gene Regulation </h4>
<div class="row">
<div style="float: left; width: 40%;">
<img style="float: left;" src="/img/research_img/normalize.PNG" width="400" height="250" hspace="20">
</div>
<div style="float: right; width: 50%;">
<font size="2">
GEO Database has massive public available gene expression data, while the chromatin accessibility data (e.g. DNase-seq) is quite limited. This study aims to build a scalable and computational efficient regression model using matched gene expression data and regulatory profiles, generalize our model for prediction across multiple platforms, and with the ultimate goal to construct comprehensive regulome landscape.  
</font>
</div>
</div>

<h4> Microbiome and Metagenomics </h4>
<div class="row">
<div style="float: left; width: 40%;">
<img style="float: left;" src="/img/research_img/qqplot.png" width="400" height="250" hspace="20">
</div>
<div style="float: right; width: 50%;">
<font size="2">
Analyzing human microbiome data from individual study might yield inconsistent results due to technical variabilities. To address the potential biases from sequencing protocols, we developed a kernel regression framework for integrative analysis of multiple microbiome datasets, which accounts for between-study dissimilarity, and proposed a score based statistic for joint test of common effect and heterogeneity.
</font>
</div>
</div>

<br>
<h3 style="color:#3182bd"> Biomedical Application and Collaboration Projects </h3>


<h4> Cancer Immunology </h4>
<div class="row">
<div style="float: left; width: 40%;">
<img style="float: left;" src="/img/research_img/circlize.PNG" width="400" height="250" hspace="20">
</div>
<div style="float: right; width: 50%;">
<font size="2">
Immune checkpoint inhibitors are immunotherapy drugs that block checkpoint protein from binding with their partner proteins (e.g. PD-1 and PD-L1). We are interested in detecting such ligand and receptor pair and their interaction between different cell subpopulation based on scRNA-seq data.
</font>
</div>
</div>

<h4> Human Microbiome Project Data </h4>
<div class="row">
<div style="float: left; width: 40%;">
<img style="float: left;" src="/img/research_img/hmpdacc.PNG" width="400" height="250" hspace="20">
</div>
<div style="float: right; width: 50%;">
<font size="2">
Human Microbiome Project data can be accessed via The database of Genotypes and Phenotypes (dbGaP). Our contribution is to develop open source R toolkit to download, decrypt and analyze controlled access data for users with dbGaP repository key.

</font>
</div>
</div>

<br>
<h3 style="color:#3182bd"> Course Projects </h3>
- BARTlearner: A unified software for heterogenous treatment effect estimation in observation studies via BART. [[link]](https://github.com/stephlee3/BARTlearner) <br/>
_Special Studies and Research Biostatistics_ | Advisor: [Ravi Varadhan](https://coah.jhu.edu/people/Faculty_personal_Pages/Varadhan.html) |PH.140.840

- A Bayesian hierarchical model for PANSS score trajectory prediction. [[link]](https://stephlee3.github.io/Bayes_hierachical_special_studies/) <br/>
_Advanced Topics in Bayesian Hierarchical Models_ | PH.140.850

- NHANES data presentaion. [[link]](https://stephlee3.github.io/Wearable_special_studies/) <br/>
_Advanced Topics in Wearable Computing_ | PH.140.850

- 2019-2020 NBA Playoffs Prediction. [[link]](https://advds71x.github.io/NBAproj/) <br/>
_Advanced Data Science_ | PH.140.712
